### ISSUE TITLE
installation guide need to be described at Getting Started docs

### ISSUE THREAD

**Issue by Einere** [dooboolab/dooboo-ui#98]
**Is your feature request related to a problem? Please describe.**

currently, there is no installation guide in [Getting Started](https://dooboo-ui.dooboolab.com/?path=/docs/overview-getting-started--page).
and, contributing guide need to move at [Contributing](https://dooboo-ui.dooboolab.com/?path=/docs/contributing-contributing--page).

**Describe the solution you'd like**

move contributing section from Getting Started to Contributing.
add installation guide at Getting Started.

**Describe alternatives you've considered**

**Additional context**

If maintainer allow, i will do this documentation!

**Comment by hyochan** [comment]
Thanks for the issue. I think you'd want to consider providing installation guide for users and not contributors.
If I am understanding correctly, it would be great to contribute!

**Comment by Einere** [comment]
yes, i thinks `Get Started` section is for beginners. not contributors.
I will PR for documentation soon! 👏

**Comment by luke-hanwook** [comment]
@Einere 
I think we have to remember dooboo-ui is universal supprted.
so installation guide need to include a various environment setup documents. (Although i think it is not needed yet...)
ex. for web user, cli quickstart, native cli setup... and so on..

I found reference site for share
https://docs.nativebase.io/installation
https://reactnativeelements.com/docs/web_usage
https://reactnative.dev/docs/environment-setup

And i have an experience to install dooboo-ui on various environment.
so i think i can help you for many problem on installation.

**Comment by hyochan** [comment]
@Einere @luke-hanwook What a great discussion. Looking forward to your collaborations 🥇

**Comment by luke-hanwook** [comment]
@hyochan 
I tried to resolve problem to install "dooboo-ui" on various environment but there are many unresolved problem.
I organized many problem things and i will share to him.
It's not awesome thing. but i think we will enhance every problem and i will try continuously 😺

**Comment by Einere** [comment]
@luke-hanwook 
i commited `docs/installation-guide` to my dooboo-ui repo.
would you review or contribute at that repo? 👏
